src/game/options_menu.c

```diff
@@ -604,6 +604,46 @@ void optmenu_check_buttons(void) {
             #endif
             optmenu_opt_change(&currentMenu->opts[currentMenu->select], 0);
         }
+     } else if (gPlayer1Controller->buttonPressed & D_CBUTTONS) {
```

Comment:
not my branch, but I'd suggest combining the U/D with the stickY, and L/R with stickX.

something like

```if (ABS(gPlayer1Controller->stickY) > 60 || (gPlayer1Controller->buttonPressed & (D_CBUTTONS|U_CBUTTONS)) { ```

as you're copying/pasting a lot of common code this way

Reply:
I'm not really a programmer and really wanted this to work, so just copy pasted to do it myself.  Of course if there is a cleaner way to implement the same functionality that is great.  Just hope it gets committed in any form!

Reply:
https://github.com/sm64pc/sm64ex/pull/435

I have updated and resubmitted this.  Not sure how to do things completely right with github so going to cancel this one.